Test/Variational/test_variational_mixed.py

```diff
@@ -83,7 +83,7 @@ def test_n_samples_api(vstate):
     with raises(
         ValueError,
     ):
-        vstate.n_discard = -1
+        vstate.n_discard_per_chain = -1
```

Comment:
isn't the default value ```None```?

Reply:
this test just checks that it errors if you supply a negative value. see the `with raises()` above

Reply:
oh I see sorry